tests/unit/sagemaker/serverless/test_model.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2017-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+from mock import Mock
+import pytest
+
+from sagemaker.serverless import LambdaModel
+
+IMAGE_URI = "123456789012.dkr.ecr.us-west-2.amazonaws.com/my-lambda-image:latest"
+ROLE = "arn:aws:iam::123456789012:role/MyLambdaExecutionRole"
+
+
+@pytest.fixture
+def mock_client():
+    return Mock()
+
+
+@pytest.mark.parametrize("wait", [False, True])
```

Comment:
How about adding a test for when the deployment fails?

Reply:
Good idea. I'll do so

Reply:
Addressed